css/custom.css

```diff
@@ -3,6 +3,10 @@
 	width: 50%;
 }
 
+body.noscroll {
+    overflow: hidden;

```

Comment:
#Is hiding the scrollbar necessary? Removing this would likely solve the issue mentioned by @wongrm here https://github.com/pspc-spac/avadepth/pull/5#pullrequestreview-651690102#

Reply:
Likewise there is also a scrollbar here that can likely be hidden by editing the css.
![image](https://user-images.githubusercontent.com/23072744/117083237-1723fa00-acf9-11eb-8958-e0ea97bb6f6c.png)

Reply:
re: Scrollbar overflow. The overflow option was one of the only ways to actually prevent the page from scrolling when the cursor was over the map. I tried playing with event capturing and bubbling and could never get the event to prevent default behaviour. I can look into it further if it is a requirement.

Reply:
re: Scrollbar on iframe. I have no experienced this. Which browser are you using?

Reply:
> Likewise there is also a scrollbar here that can likely be hidden by editing the css.
> 

Is the scroll bar appearing on the map for just the cbw?  I think this might now be an isolated issue.  Might be out of scope for this PR